### PR TITLE
[OUT-260] [OUT-261] Fixed, print and line lengths

### PR DESCRIPTION
**Pull request by naseberry**
* [OUT-260] Updated line lengths
* [OUT-261] Fixed print utility flex-basis width issue on IE

### CODE REVIEW

File: src/stylesheets/elements/_headings.scss [hunk 1]
```diff
@@ -147,6 +135,15 @@ footer h2 {
   }
 }
 
+.hero h1 {
+  @include media($tablet) {
```
Comment (jakemulley) [inline src/stylesheets/elements/_headings.scss]: Do you want to `@include line-length` here?

File: src/stylesheets/components/_article.scss [hunk 2]
```diff
@@ -8,8 +8,18 @@
 article {
   margin: $base-spacing-unit-extra-large 0 $base-spacing-unit-small;
 
+  > * {
```
Comment (jakemulley) [inline src/stylesheets/components/_article.scss]: Can we move this below the `@include media($desktop)` line? > * is for child elements, but the mixin affects the `article` element

File: src/stylesheets/elements/_headings.scss (hunk 1 above)
Comment (naseberry) [reply src/stylesheets/elements/_headings.scss]: No, different values, at different break-points

Comment (jakemulley) [reply src/stylesheets/elements/_headings.scss]: My bad...

**Pull request merged**